src/ripple/core/impl/Config.cpp

```diff
@@ -411,26 +413,22 @@ void Config::loadFromString (std::string const& fileContents)
     if (getSingleSection (secConfig, SECTION_SSL_VERIFY, strTemp, j_))
         SSL_VERIFY          = beast::lexicalCastThrow <bool> (strTemp);
 
-    if (getSingleSection (secConfig, SECTION_VALIDATION_SEED, strTemp, j_))
+    if (getSingleSection (secConfig, SECTION_VALIDATION_SEED, VALIDATION_SEED, j_))
```

Comment:
My thinking is if the config file doesn't have a valid section for a variable, we should clear that value, not leave it at it's current value. But it probably isn't needed the way we use this code, and the current code matches the old behaviour. Yeah, keep as is.